neoexchange/core/management/commands/lightcurve_extraction.py

```diff
@@ -355,16 +360,16 @@ def handle(self, *args, **options):
             self.stdout.write("Error determining telescope diameter, assuming 0.4m")
             tel_diameter = 0.4*u.m
 
-        # Create, name, open ALCDEF file.
-        if obs_date:
-            alcdef_date = options['date']
-        else:
-            alcdef_date = sb_day
-        base_name = '{}_{}_{}_{}_'.format(obj_name, sb_site, alcdef_date, start_super_block.tracking_number)
-        alcdef_filename = os.path.join(datadir, base_name + 'ALCDEF.txt')
-        output_file_list.append('{},{}'.format(alcdef_filename, datadir.lstrip(out_path)))
-        alcdef_file = default_storage.open(alcdef_filename, 'w')
         for super_block in super_blocks:
+            # Create, name, open ALCDEF file.
+            if obs_date:
+                alcdef_date = options['date']
+            else:
+                alcdef_date = super_block.block_start.strftime("%Y%m%d")
+            base_name = '{}_{}_{}_{}_'.format(obj_name, super_block.get_sites().replace(',', ''), alcdef_date, super_block.tracking_number)
+            alcdef_filename = base_name + 'ALCDEF.txt'
+            output_file_list.append('{},{}'.format(alcdef_filename, datadir.lstrip(out_path)))
+            alcdef_file = tempfile.NamedTemporaryFile('w+')  # default_storage.open(alcdef_filename, 'w')
```

Comment:
I'm not sure this line does anything at the moment

Reply:
You're right. I've nuked it.